core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/test-constants.ts

```diff
@@ -40,7 +40,7 @@ export default {
     ExternalLink61MillionChildren:
       'https://uis.unesco.org/en/news/world-poverty-could-be-cut-half-if-all-adults-completed-secondary-education',
     ExternalLinkEvenThoseWhoAreInSchool:
-      'https://uis.unesco.org/sites/default/files/documents/fs46-more-than-half-children-not-learning-en-2017.pdf',
+      'http://uis.unesco.org/sites/default/files/documents/fs46-more-than-half-children-not-learning-en-2017.pdf',
```

Comment:
Why change this? We prefer https links generally.

Reply:
@seanlip, the reason for this is to ensure that the href matches the expectedUrl. The href of the button, which is embedded in the i18n files, does not contain an ‘s’ in http. However, we do have ‘https’ above. To match them, I need to adjust one of them. As you suggested, we prefer not to make changes in i18n, so I made the adjustment here.

Reply:
I see. OK in this case, make it https everywhere. Better to send our users to secure links.

The note about not making changes in i18n files is to avoid adding more complex elements to i18n files. http --> https does not increase complexity.

Reply:
Got that. sure.